symfony/plugins/orangehrmTimePlugin/config/routes.yaml

```
@@ -138,6 +138,23 @@ apiv2_time_my_timesheet_time_format_config:
     id: 0
     _api: OrangeHRM\Time\Api\TimesheetTimeFormatAPI
 
+apiv2_time_copy_activity:
+  path: /api/v2/time/projects/{toProjectId}/activities/copy
```

I think we may need `fromProjectId` for validation. What ever the given activity ids we have to check, those ids belongs to this `fromProjectId`.

```suggestion
  path: /api/v2/time/projects/{toProjectId}/activities/copy/{fromProjectId}
```

---

done